web/src/main/webapp/js/main.js

```diff
@@ -1070,15 +1080,19 @@ function routeLatLng(request, doQuery) {
 
             var hiddenDiv = $("<div id='routeDetails'/>");
             hiddenDiv.hide();
+            var versionInfoHiddenDiv = $("<div id='versionDetails'/>");
+            versionInfoHiddenDiv.hide();
```

Comment:
Add the version details to the hiddenDiv, then no extra hide() and toggle() call is necessary